### ISSUE TITLE
Option to enable targeting OTP apps

### ISSUE THREAD

**Issue by juhlig** [ankhers/havoc#13]
We would like to see the effects on `ranch` when wreaking havoc in the `ssl` application. This is not possible, however, since in 4577100aacff0a52762309b242700c5b06aa6379, killing processes from OTP applications has been invariably disabled in `havoc`. I wonder, for what reason?

It is not possible to re-enable them by including them in the `applications` option, either. Even if, eg by changing https://github.com/ankhers/havoc/blob/e7701299922f7ac5da51d8ed50b51141321f50d9/src/havoc.erl#L234 to something like
```erlang
andalso not(lists:member(App, ?OTP_APPS -- State#state.applications))
```
this would
1) imply targeting _only_ those applications, and
2) be impractical if you wanted to include (almost) all OTP applications (we only want to exclude `kernel`, `common_test` and `sasl`)

**Comment by ankhers** [comment]
I initially disabled killing OTP applications because havoc was initially built to kill your own applications. I had not thought about the usecase of seeing how things react if OTP applications were to go down. I would be happy to do something like this.

**Comment by ankhers** [comment]
Fixed in 5440cde5fdad41d6f956d99811eff6a6121738d9. There is now the `otp_applications` option for this.
